src/Panel.vue

```diff
@@ -30,7 +28,7 @@
                                       @click.native.stop.prevent="expand()"
                                       @is-open-event="retrieveOnOpen"></panel-switch>
                         <button type="button" class="close-button btn btn-default"
```

Comment:
Migration guide suggest that we change `btn-default` to `btn-secondary`. This change is applicable to `PanelSwitch.vue` as well.